mvrx/src/main/kotlin/com/airbnb/mvrx/BaseMvRxViewModel.kt

```diff
@@ -33,6 +33,7 @@ abstract class BaseMvRxViewModel<S : MvRxState>(
     private val stateStore: MvRxStateStore<S> = RealMvRxStateStore(initialState)
 ) : ViewModel() {
     private val debugMode = if (MvRxTestOverrides.FORCE_DEBUG == null) debugMode else MvRxTestOverrides.FORCE_DEBUG
+    private val forceUserTestObserver = MvRxTestOverrides.FORCE_USE_TEST_OBSERVER
```

Comment:
typo: `forceUseTestObserver`

Reply:
Actually, I was thinking to rename to `forceDisableLifecycleAwareObserver` and flag to `FORCE_DISABLE_LIFECYCLE_AWARE_OBSERVER` . WYT?